MDSassignment/guestbook/post/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.utils import timezone
+
+
+STATUS_CHOICES = (
+	('h', 'Hidden'),
+	('p', 'Published'),
+	)
+
+class Post(models.Model):
+	name = models.CharField(max_length=150)
+	text = models.TextField()
+	email = models.EmailField(max_length=254, blank=True, null=True)
+	time = models.DateTimeField(default=timezone.now)
+	status = models.CharField(max_length=1, choices=STATUS_CHOICES, default='p')	
```

Comment:
Have you thought of using a `BoolenField` instead? E.g. `is_public = models.BooleanField(default=True)`?